content/Product/OpenshiftMonitoring/faq.md

```diff
@@ -0,0 +1,140 @@
+# Debugging the in-cluster monitoring stack
+
+## Overview
+
+This serves as a collection of resources that relate to FAQ around configuring/debugging the in-cluster monitoring stack.
+
+## Identify the cause of some common issues
+
+### ServiceMonitor created but no metrics from the service
+
```

Comment:
```suggestion

Applicable versions: OpenShift 4.x etc...

```

Reply:
The same ideally for other guides

Reply:
Sure, but some problems & solution might be adequate to subset of versions. If all, for now, then fine